source/overloading.tex

```diff
@@ -3506,6 +3506,17 @@
 the rules in~\ref{over.match.oper} determine which, if any, interpretation is
 used.
 
+\pnum
+A \defn{comparison function} is an operator function
+for one of the following binary operators:
+\tcode{<},
+\tcode{<=},
+\tcode{>},
+\tcode{<=},
```

Comment:
This is the same one as 2 lines up :)

Reply:
FWIW, the canonical ordering used for these operators in the library is:
* `==`
* `!=`
* `<`
* `>`
* `<=`
* `>=`

Reply:
Fixed.